manifests/vars-template.yml

```diff
@@ -1,10 +1,10 @@
 ---
 
-pcap_server_api:
+pcap_api:
   vm_type: default
   networks: [{name: default}]
   cf_api: https://api.cf.cfapp.com
-  ca_path: /cf/pcap_server_mtls_ca
+  ca_path: /cf/pcap_mtls_ca
```

Comment:
you've renamed this CA to `pcap_agent_mtls_ca` above, so this won't work.
however, I like `pcap_mtls_ca` more, because it makes it clear that the CA is shared among both pcap-api and pcap-agent, not just for the agent.

So I would suggest to rename the ca to `pcap_mtls_ca` also in the ops file.